lib/Doctrine/Common/Annotations/AnnotationReader.php

```diff
@@ -184,6 +184,14 @@ public function __construct()
     }
 
     /**
+     * @param bool $notImportedAnnotationsIgnored
+     */
+    public function setParserIgnoreNotImportedAnnotations($notImportedAnnotationsIgnored)
```

Comment:
Great idea! I just wanted to prepare changes as simple as possible without impact to whole code base. The best solution would be (of course) to add possibility to inject AnnotationReader::$preParser and AnnotationReader::$parser by passing them to constructor.
Don't you think?